baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/StrictUnusedVariableTest.java

```diff
@@ -133,63 +133,49 @@ void handles_lambdas_in_static_init() {
 
     @Test
     public void renames_previous_suppression() {
-        refactoringTestHelper
-                .addInputLines(
+        // if it is prefixed with 'unused' don't do anything.
```

Comment:
This is not the current functionality, but something I was thinking about. It's could be a bit annoying if we show this warning when the user already acknowledged that the variable is unused by prefixing it with `unused` or `ignored`.